plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -210,6 +210,8 @@ func init() {
 		Rules: []rbac.PolicyRule{
 			rbac.NewRule("get", "list", "watch", "update").Groups(extensionsGroup).Resources("replicasets").RuleOrDie(),
 			rbac.NewRule("update").Groups(extensionsGroup).Resources("replicasets/status").RuleOrDie(),
+			rbac.NewRule("get", "list", "watch", "update").Groups(appsGroup).Resources("replicasets").RuleOrDie(),
```

Comment:
No need to add 2 new Rules, just amend L211-212 to use `.Groups(extensionsGroup ,appsGroup)` instead, since the rules are exactly the same.